modules/security-context-constraints-psa-opting.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-opting_{context}"]
+= Controlling pod security admission synchronization
+
+You can enable or disable automatic pod security admission synchronization for most namespaces.
```

Comment:
We need to make clear in the docs that is NOT possible to enable the synchronization for the ns which are part of the OCP payload: 

https://github.com/openshift/cluster-policy-controller/blob/c7201edacb6e4ac1f8a1624b152911f3c404dbc7/pkg/psalabelsyncer/podsecurity_label_sync_controller.go#L42-L111

The docs should contain the list or a link for the implementation ( release tag 4.11 ) : https://github.com/openshift/cluster-policy-controller/blob/release-4.11/pkg/psalabelsyncer/podsecurity_label_sync_controller.go#L42-L111

Reply:
I wanted to specifically leave this vague in case it changes